### DISCUSSION TITLE
is it possible to add a request body, params, or other custom info somewhere in the report?

### DISCUSSION THREAD

**Discussion by pargevghazaryan** [ludeknovy/jtl-reporter#245] [Q&A]
is it possible to add a request body, params, or other custom info somewhere in the report?

**Comment by ludeknovy** [comment]
Hi @pargevghazaryan! There’s possibility to add a note to the report.

**Reply by pargevghazaryan** [reply]
@ludeknovy thanks for the response. but i'd like to see data for requests. 
in my case the most requests are graphql and I have set graphql operation name as labels. but I cant see what is exact variables for the request that has low performance.

**Reply by ludeknovy** [reply]
I see, but that’s not possible, because all the  aggregations are per label name. 
And if I am not mistaken, the specific param inputs are not stored in the csv file

**Reply by pargevghazaryan** [reply]
what about if I store it in the request "context"?

**Reply by ludeknovy** [reply]
That won't help. The only way how to display it currently is to use the parameters in the label name.
